uitools/common/CMakeLists.txt

```diff
@@ -0,0 +1,146 @@
+# Copyright 2012-2025 Esri
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+cmake_minimum_required(VERSION 3.10)
```

Comment:
minimal version is 3.10 since 3.31. 3.10 was released in 2017.

Reply:
I think we should go higher.

https://doc.qt.io/qt-6/cmake-supported-cmake-versions.html 6.8 lists 3.16 as the minimum, so I would go with 3.16.